app/models/post.rb

```diff
@@ -0,0 +1,20 @@
+class Post < ApplicationRecord
+  belongs_to :author , class_name: "User"
+  has_many :comments
+  has_many :likes
+
+  after_save :update_posts_counter
+
+  def update_posts_counter
+    if author.posts_counter
+      author.posts_counter += 1
+    else 
+      author.posts_counter = 1
+    end
+    author.save
+  end
```

Comment:
- [ ] **[OPTIONAL]** I think we can save some code here.

  ```
  def update_posts_counter
    author.increment!(:posts_counter)
  end
  ```

Reply:
Thank you for the suggestion. I implemented it but in the end, the posts_counter was not incrementing, it remained 1 despite adding more comments (refer to the image below). Is there a way that it could effectively increment using that method?

![image](https://user-images.githubusercontent.com/59999191/190833200-ee27881f-ea49-47be-b8f0-498d8fbb3ef4.png)